config/locations/holding_locations.json

```diff
@@ -355,6 +355,26 @@
         "holding_library": null,
         "delivery_locations": []
     },
+    {
+        "label": "School of Architecture Faculty Publications",
+        "code": "arch$fac",
+        "aeon_location": false,
+        "recap_electronic_delivery_location": false,
+        "open": true,
+        "requestable": true,
+        "always_requestable": false,
+        "circulates": true,
+        "remote_storage": "",
+        "fulfillment_unit": "General",
+        "url": "https://bibdata.princeton.edu/locations/holding_locations/arch$fac.json",
+        "library": {
+            "label": "Architecture Library",
+            "code": "arch",
+            "order": 0
+        },
+        "holding_library": null,
+        "delivery_locations": []
```

Comment:
@mzelesky @kevinreiss delivery locations?

Reply:
It should be the architecture library only. These materials are on-site.